react/features/welcome/components/LocalVideoTrackUnderlay.native.js

```diff
@@ -91,9 +92,9 @@ class LocalVideoTrackUnderlay extends Component<*, *> {
         return (
             <View style = { this.state.style }>
                 <VideoTrack videoTrack = { this.props._localVideoTrack } />
-                <View style = { styles.localVideoTrackOverlay }>
+                <TintedView>
```

Comment:
Is TintedView transparent by default ?

Reply:
No, it uses a background color with a certain opacity. The defaults are (almost) black and 0.8.